rules/examples/Invoice reponse use cases/T111-uc009-Partial payment has been initiated.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+                Content:
+                This file contains a PEPPOL Invoice Message Response, profile 63.
+				It demonstrates Use Case 9 — Invoice is partially paid. 
+				
+				Use case shows where a buyer initiates a partial payment towards the accepted amount.		
+				
+                Errors:
+                None
+                
+                Warnings:
+                None
+				
+-->
+<ApplicationResponse xmlns="urn:oasis:names:specification:ubl:schema:xsd:ApplicationResponse-2" xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2" xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<cbc:CustomizationID>urn:fdc:peppol.eu:poacc:trns:invoice_response:3</cbc:CustomizationID>
+	<cbc:ProfileID>urn:fdc:peppol.eu:poacc:bis:invoice_response:3</cbc:ProfileID>
+	<cbc:ID>imrid001</cbc:ID>
+	<cbc:IssueDate>2017-12-01</cbc:IssueDate>
+	<cac:SenderParty>
+		<cbc:EndpointID schemeID="0196">6963495890</cbc:EndpointID>
+		<cac:PartyLegalEntity>
+			<cbc:RegistrationName>Invoice processing service</cbc:RegistrationName>
+		</cac:PartyLegalEntity>
+	</cac:SenderParty>
+	<cac:ReceiverParty>
+		<cbc:EndpointID schemeID="0196">6841569459</cbc:EndpointID>
+		<cac:PartyLegalEntity>
+			<cbc:RegistrationName>Seller company</cbc:RegistrationName>
+		</cac:PartyLegalEntity>
+	</cac:ReceiverParty>
+	<cac:DocumentResponse>
+		<cac:Response>
+			<cbc:ResponseCode listID="UNCL4343OpSubset">AP</cbc:ResponseCode>
```

Comment:
The Response Code appears to be wrong, but the Schematron seem to allow it. Either fix the specification or fix the schematrons

Reply:
Changed as proposed